### PR TITLE
Field docstring refactor (2/2)

### PR DESCRIPTION
**Pull request by matthew7838**
### Summary
Wraps up all the docstring updates.

### CODE REVIEW

**Review by allisterakun — APPROVED** [review]
Great job! Thanks you the hard work, Matthew!

File: RUFAS/routines/field/soil/soil.py [hunk 1]
```diff
@@ -16,48 +16,60 @@
...
 
         # Process components
         self.soil_temp = SoilTemp(self.data)
-        """Process component that tracks and updates the temperatures within the soil profile"""
         self.phosphorus_cycling = PhosphorusCycling(self.data)
-        """Process component managing phosphorus on top of and in the soil profile"""
         self.carbon_cycling = CarbonCycling(self.data)
-        """Process component that handles carbon cycling (through decomposition) in the soil."""
         # TODO: need to add phosphorus, manure, and carbon cycling main methods methods to the soil methods.
         #   It is unclear to me how best to do that.
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/soil.py]:
```suggestion
```

File: RUFAS/routines/field/soil/soil.py [hunk 2]
```diff
@@ -16,48 +16,60 @@
...
+        Process component that tracks snow.
 
-        """
+    """
+
+    def __init__(self, soil_data: Optional[SoilData] = None, field_size: Optional[float] = None):
         self.data = soil_data or SoilData(field_size=field_size)
-        """object that tracks all soil variable throughout the simulation"""
 
         # Process components
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/soil.py]:
```suggestion
```

File: RUFAS/routines/field/soil/soil.py [hunk 3]
```diff
@@ -16,48 +16,60 @@
...
         self.phosphorus_cycling = PhosphorusCycling(self.data)
-        """Process component managing phosphorus on top of and in the soil profile"""
         self.carbon_cycling = CarbonCycling(self.data)
-        """Process component that handles carbon cycling (through decomposition) in the soil."""
         # TODO: need to add phosphorus, manure, and carbon cycling main methods methods to the soil methods.
         #   It is unclear to me how best to do that.
         self.nitrogen_cycling = NitrogenCycling(self.data)
-        """Process component for managing nitrogen within the soil profile."""
 
         # Water components
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/soil.py]:
```suggestion
```

File: RUFAS/routines/field/soil/soil_config_factory.py [hunk 4]
```diff
@@ -24,20 +25,21 @@ def create_soil_data(
...
 
         Parameters
         ----------
         field_size : float
             Size of the field. Used to initialize a Soil object for this module to work with, if a pre-configured
-            SoilData object is not provided (ha)
+            SoilData object is not provided (ha).
         config : SoilConfiguration
-            configuration of the soil
+            configuration of the soil.
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/soil_config_factory.py]:
```suggestion
            Configuration of the soil.
```

File: RUFAS/routines/field/soil/soil_data.py [hunk 5]
```diff
@@ -344,29 +421,33 @@ def _subdivide_top_layer(self, field_size: float) -> None:
...
     def get_vectorized_layer_attribute(self, attribute: str) -> List[any]:
-        """returns a list containing the specified attribute for each soil layer
+        """
+        Returns a list containing the specified attribute for each soil layer.
 
         Parameters
         ----------
         attribute : str
-            the LayerData attribute or property to be vectorized
+            the LayerData attribute or property to be vectorized.
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/soil_data.py]:
```suggestion
            The LayerData attribute or property to be vectorized.
```

File: RUFAS/routines/field/soil/soil_data.py [hunk 6]
```diff
@@ -344,29 +421,33 @@ def _subdivide_top_layer(self, field_size: float) -> None:
...
 
         Returns
         -------
         layered_attribute : list[any]
-            values of the specified attribute for each layer
+            values of the specified attribute for each layer.
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/soil_data.py]:
```suggestion
            Values of the specified attribute for each layer.
```

File: RUFAS/routines/field/soil/soil_data.py [hunk 7]
```diff
@@ -344,29 +421,33 @@ def _subdivide_top_layer(self, field_size: float) -> None:
...
     def set_vectorized_layer_attribute(self, attribute: str, values: List[any]) -> None:
-        """sets a given attribute for all layers in soil_data
+        """
+        Sets a given attribute for all layers in soil_data.
 
         Parameters
         ----------
         attribute : str
-            the LayerData attribute to set
+            the LayerData attribute to set.
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/soil_data.py]:
```suggestion
            The LayerData attribute to set.
```

File: RUFAS/routines/field/soil/soil_data.py [hunk 8]
```diff
@@ -344,29 +421,33 @@ def _subdivide_top_layer(self, field_size: float) -> None:
...
         values : list[any]
-            values of the attribute to set for each layer
+            values of the attribute to set for each layer.
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/soil_data.py]:
```suggestion
            Values of the attribute to set for each layer.
```

**Review by Isgotkowitz — APPROVED** [review]
Noticed a few small things that need to be corrected, but overall looks good. Nice job Matthew.

**Pull request merged**